hwy/ops/shared-inl.h

```diff
@@ -200,13 +200,22 @@ HWY_API void CopyBytes(const From* from, To* to) {
 #endif
 }
 
-HWY_API size_t PopCount(const uint64_t x) {
+HWY_API size_t PopCount(uint64_t x) {
 #if HWY_COMPILER_CLANG || HWY_COMPILER_GCC
   return static_cast<size_t>(__builtin_popcountll(x));
-#elif HWY_COMPILER_MSVC
+#elif HWY_COMPILER_MSVC && _WIN64
```

Comment:
Let's use HWY_COMPILER_MSVC && HWY_ARCH_X86_64?

Reply:
Did not know that existed. Made the change as you suggested.